spec.md

```diff
@@ -2103,6 +2219,47 @@ Condition | gRPC Code | Description | Recovery Behavior
 | Call not implemented | 12 UNIMPLEMENTED | NodeGetInfo call is not implemented by the plugin or disabled in the Plugin's current mode of operation. | Caller MUST NOT retry. Caller MAY call `ControllerGetCapabilities` or `NodeGetCapabilities` to discover Plugin capabilities. |
 
 
+#### `NodeExpandVolume`
+
+A Node Plugin MUST implement this RPC call if it has `EXPAND_VOLUME` node capability.
+This RPC call allows CO to expand volume on a node.
+
+`NodeExpandVolume` ONLY supports expansion of already node-published or node-staged volumes on the given `volume_path`.
+
+If plugin has `STAGE_UNSTAGE_VOLUME` node capability - `NodeExpandVolume` MUST be called after successful `NodeStageVolume`.
+`NodeExpandVolume` MAY be called before or after `NodePublishVolume`.
```

Comment:
When will `NodeExpandVolume` be called if plugin has no `STAGE_UNSTAGE_VOLUME` node capability? These plugins(such as `ceph rbd`) map/mount the volume devices to the nodes inside `NodePublishVolume`, so maybe we need to make sure `NodeExpandVolume` called after `NodePublishVolume` for these plugins.

Reply:
Yeah - I thought that was clear but I will clarify it.